### DISCUSSION TITLE
`CompatRoute` doesn't render at all

### DISCUSSION THREAD

**Discussion by RedHatter** [remix-run/react-router#11351] [v5 to v6 Migration]
I'm trying to use `react-router-dom-v5-compat` to upgrade from v5 to v6 however I can't get any `CompatRoute` to render. Below I've included the simplest usage of `CompatRoute` trying to narrow down the problem.
```typescript
import type { FC } from 'react';
import { BrowserRouter, Switch } from 'react-router-dom';
import { CompatRoute, CompatRouter } from 'react-router-dom-v5-compat';

const About: FC = () => <h1>About</h1>;

const App: FC = () => (
  <BrowserRouter>
    <CompatRouter>
      <Switch>
        <CompatRoute path="/about" element={<About />} />
      </Switch>
    </CompatRouter>
  </BrowserRouter>
);
```
The about route above does not render. Am I missing something? I have looked over the docs, migration guide, and everything else I could find but I don't see any reason this shouldn't work.

[sandbox link](https://stackblitz.com/edit/github-ws5ytb?file=src%2FApp.tsx)

**Comment by SamSverko** [comment]
Looks like changing the `element` prop to `component` works for me:

```tsx
<CompatRoute path="/about" component={About} />
```

Results:

![Screenshot 2024-03-20 at 4 32 59 PM](https://github.com/remix-run/react-router/assets/4431967/4c6a7726-7a0d-4327-baa0-716d887ebace)

> [!IMPORTANT]
> Don't forget to visit the `/about` route (i.e. "https://sandbox-url-stuff.webcontainer.io/about") in the sandbox's URL! Your sandbox links to the index page - which has a path of `/`.
